.buildbot.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 CHERIBUILD=~/build
```

Comment:
@jacobbramley Had an issue with expanding `~`, so I would maybe hardcode this for now to be safe, or change to `$HOME`. 

If this is meant to be the path to a `cheribuild` checkout, that is `$HOME/cheri-build`. `$HOME/build` is the current working directory, and a checkout of the tested (i.e., this) repo.

Reply:
I would also perhaps change these to conditional variables, only set if they are not. For example, I might plan to provide a `$CHERIBUILD` variable in the image, and `$PYTHONPATH` is already provided (in case I'd like to move the `cheribuild` clone).

Reply:
done in   [9621e1f](https://github.com/capablevms/bdwgc/pull/17/commits/9621e1f11c82b3e3eee98f8b42a016e55193c1be)